pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -126,27 +161,30 @@ in stdenv.mkDerivation rec {
   PARQUET_TEST_DATA =
     if doInstallCheck then "${parquet-testing}/data" else null;
   GTEST_FILTER =
-    if doInstallCheck then let
-      # Upstream Issue: https://issues.apache.org/jira/browse/ARROW-11398
-      filteredTests = lib.optionals stdenv.hostPlatform.isAarch64 [
-        "TestFilterKernelWithNumeric/3.CompareArrayAndFilterRandomNumeric"
-        "TestFilterKernelWithNumeric/7.CompareArrayAndFilterRandomNumeric"
-        "TestCompareKernel.PrimitiveRandomTests"
-      ];
-    in "-${builtins.concatStringsSep ":" filteredTests}" else null;
+    if doInstallCheck then
```

Comment:
This should use some lib.optional variant.

Reply:
Done.